packages/codebytes/src/libs/eventTracking.ts

```diff
@@ -0,0 +1,16 @@
+import { createTracker } from '@codecademy/tracking';
+
+const IS_DEV = process.env.NODE_ENV === 'development';
```

Comment:
do we have NODE_ENV set in this repo?

Reply:
We wouldn't quite need to set this here, as it will be coming from the host (portal-app or the monolith, or whatever calls codebytes). A quick way to see that it exists, is to console.log the `process.env.NODE_ENV` value in storybook and see that it is development. 

<img width="1437" alt="Screen Shot 2022-01-23 at 1 45 27 PM" src="https://user-images.githubusercontent.com/4481281/150693688-7a5e7728-7ab6-4d13-9115-536f985bcb91.png">

Reply:
(It should work out of the box)

Reply:
Confirmed is working via console log: <img width="517" alt="Screen Shot 2022-01-24 at 2 13 43 PM" src="https://user-images.githubusercontent.com/12686946/150848788-d4db6166-4b28-4ee1-9c15-c5990e83ad7a.png">